components/WorkScheduleForm/WorkScheduleForm.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import { ScheduleFormContainer } from "./WorkScheduleForm.styled";
+import DatePickerRange from "../DatePicker/DatePickerRange";
+import { Botton } from "./WorkScheduleForm.styled";
```

Comment:
Typo here as well. Supposed to be called "Button" here and the file where it's defined.